Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/Pointers/SpherePointerVisual.cs

```diff
@@ -82,6 +77,11 @@ private void CheckAsset(object asset, string fieldname)
 
         public void Update()
```

Comment:
Do you know if there's a reason all the MonoBehaviour methods are `public` in this class?

Reply:
I don't. I'd prefer if they were private.